sceptre/stack.py

```diff
@@ -186,6 +171,8 @@ def __init__(
         self.profile = profile
         self.template_key_prefix = template_key_prefix
         self.iam_role_session_duration = iam_role_session_duration
+        self.ignore = ignore
+        self.obsolete = obsolete
```

Comment:
I'm not sure I understand here. Are you meaning to say we should validate here that these values are indeed booleans and not string? If the user enters `"False"`, this would actually function as _truthy_ and do the opposite of what we want. I think perhaps we blow up immediately with a helpful error if the value isn't a boolean, seeing as non-boolean values are likely to be misconfigurations.

Reply:
yes, that is exactly what I mean.  check that the user passed in strings that indicate a boolean.  I think if the user does not set to either "true" or "True" then it should be "False" (the default).   I don't think it should blow up.

Reply:
Done.